README.md

```diff
@@ -46,4 +46,4 @@ We're able to focus more on Pods when we have some cash to help keep the lights
 
 <em>Please submit all pull requests against [release/3.0](https://github.com/pods-framework/pods/tree/release/3.0) branch.</em>
 
-~Current Version: 2.5.1~
+~Current Version: 2.5.1.1~
```

Comment:
What exactly is the purpose of this line supposed to be? Current release version or current dev version? I suspect the answer is we should have both as their own line.